doc/source/reference/python_introduction.rst

```diff
@@ -50,22 +50,16 @@ signs.  Let's inaugurate the occasion appropriately -- type this::
 
 As you can see, this printed out the string "Hello, world." just as we
 expected.  Now let's try a more advanced string, one with a number in it.  For
-this we'll use the percent (``%``) operator, which is the manner by which
-values are fed into a formatted string.  We'll print pi, but only with three
-digits of accuracy.::
+this we'll use an "f-string", which is the manner by which values are fed into
+a formatted string.  We'll print pi, but only with three digits of accuracy.::
```

Comment:
Using "the" when there are at least 4 or 5 different ways to do this in Python seems exaggerated.
```suggestion
this we'll use an "f-string", which is the preferred way to format strings in modern Python.
We'll print pi, but only with three digits of accuracy.::
```